apps/modernization-ui/src/apps/landing/Layout/LoginWrapper.tsx

```diff
@@ -9,6 +9,7 @@ const LoginWrapper = ({ children }: Props) => (
     <div className={styles.layout}>
         <header>
             <img src="/nbs-logo.png" height={40} alt="" />
+            <h1 className="margin-y-0 margin-left-1">Welcome to the NBS 7 demo site</h1>
```

Comment:
suggestion:  should this be added to the `header` styling in `Layout/login-wrapper.module.scss`?